core/src/main/java/kafka/server/share/DelayedShareFetch.java

```diff
@@ -204,7 +217,29 @@ Map<TopicIdPartition, FetchRequest.PartitionData> acquirablePartitions() {
         return topicPartitionData;
     }
 
-    private void releasePartitionLocks(String groupId, Set<TopicIdPartition> topicIdPartitions) {
+    boolean isMinBytesCriteriaSatisfied(Map<TopicIdPartition, FetchRequest.PartitionData> topicPartitionData) {
```

Comment:
Should it be default scoped? If used for tests then please write // Visible for testing.